src/index.html

```diff
@@ -5,8 +5,387 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="moyo__header">
+      <a href="#" class="moyo__logo">
+        <img src="./images/logo.png" alt="Moyo logo">
+      </a>
+      <nav>
+        <ul class="moyo__list">
+          <li class="moyo__item">
+            <a
+              class="moyo__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a
              class="nav__link is-active"
              href="#"
            >
              Apple
            </a>
          </li>
```
navigation can be in separate block